src/components/common/TimePicker.tsx

```
@@ -0,0 +1,72 @@
+    document.addEventListener("click", handleClickOutside)
+    return () => {
+      document.removeEventListener("click", handleClickOutside)
+    }
+  }, [])
+
+  return (
+    <div className="relative" ref={timePickerRef}>
+      <div
+        className="flex rounded-full border border-gray-300 px-4 py-3 pl-10 focus:border-main-color focus:outline-none cursor-pointer"
+        onClick={() => setIsOpen(!isOpen)}
+      >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)